src/Controls/tests/UITests/Tests/Issues/Issue21631.cs

```diff
@@ -12,12 +12,19 @@ public class Issue21631 : _IssuesUITest
 			"Injecting base tag in Webview2 works";
 
 		[Test]
+		[Category(UITestCategories.WebView)]
 		public async Task NavigateToStringWithWebviewWorks()
 		{
 			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.iOS });
+			
+			var size = App.WaitForElement("WaitForWebView").GetRect();
+
+			for (int i = 0; i < 5 && size.Height <= 0 && size.Width <= 0; i++)
```

Comment:
This change is a bit unfortunate. I've pinged @Foda because it seems like there's a weird quirk with the WebView2 where it's not displaying when it's nested in a layout. I was trying to put it into a layout and then react of size changed but wasn't having much success.